packages/api/prisma/schema.prisma

```diff
@@ -14,3 +14,10 @@ model User {
   email    String
   image    String
 }
+
+model LearningMaterial {
+  id             Int    @id @default(autoincrement())
```

Comment:
Dajemy od razu string?

Reply:
I think it would be better to convert all IDs to strings in a separate PR so an eventual discussion about such action could take place there. For feature PR I think it's better to just follow current conventions